code/insert_between.py

```diff
@@ -0,0 +1,18 @@
+from talon import Module, actions
+
+mod = Module()
+
+@mod.action_class
+class module_actions:
+    def insert_between(before: str, after: str):
+        """Insert `before + after`, leaving cursor between `before` and `after`. Not entirely reliable if `after` contains newlines."""
+        actions.insert(before + after)
+        for _ in after: actions.edit.left()
+
+    # # This is deprecated, please use insert_between instead.
+    # def insert_cursor(text: str):
+    #     """Insert a string. Leave the cursor wherever [|] is in the text"""
+    #     if "[|]" in text:
+    #         actions.user.insert_between(*text.split("[|]", 1))
+    #     else:
+    #         actions.insert(text)
```

Comment:
Maybe leave it in but call `app.notify` to tell the user it is deprecated? That's what was done with e.g. the older Slack actions that used verb-now

Reply:
Yeah, maybe so. It looks like at least fidget is using it a fair bit in a non-Python context: https://search.talonvoice.com/search/?q=insert_cursor&fold_case=auto&regex=false&context=true

Reply:
I'll make a point of switching all of mine as soon as this goes in.